client/src/services/auth.ts

```diff
@@ -1,15 +1,79 @@
-import {NextAuthOptions} from "next-auth";
+import { NextAuthOptions, TokenSet } from "next-auth";
 import Auth0 from "next-auth/providers/auth0";
 
 export const authOptions:NextAuthOptions = {
 	session: {
-		strategy: "jwt"
+		strategy: "jwt",
+		maxAge: parseInt(process.env.AUTH_TOKEN_EXPIRATION_SECONDS!),

```

Comment:
As we discussed, env vars need abstracting through TS & Zod into a strongly typed object and `process.env` usage replacing with it.

Can we also add a lint rule to error if `process.env` is used directly?

Reply:
added zod validation. I'll look at lint rule during that issue